ext/http/00_serve.js

```diff
@@ -334,11 +337,15 @@ class CallbackContext {
   fallbackHost;
   serverRid;
   closed;
+  closing;
 
   constructor(signal, args) {
+    // The abort signal triggers a non-graceful shutdown
     signal?.addEventListener(
       "abort",
-      () => this.close(),
```

Comment:
What about `CallbackContext.close`? Won't it close server abruptly? Maybe it should be removed altogether?

Reply:
It will close the server abruptly but we only use it in the case of critical errors, but it's possible we could improve the ergonomics there. Let's punt that for later.